tensorflow_datasets/core/dataset_info.py

```diff
@@ -41,13 +41,16 @@
 import posixpath
 import pprint
 import tempfile
+import logging
 
-from absl import logging
+import absl
 import numpy as np
 import six
 import tensorflow as tf
+import tensorflow_datasets.public_api as tfds
```

Comment:
You should import individual modules, instead of the full API, otherwise it creates circular dependency: `tfds => dataset_info => tfds`.